operator/controllers/ambassador.go

```diff
@@ -115,6 +125,35 @@ func getAmbassadorCircuitBreakerConfig(
 	return nil, nil
 }
 
+// Return TLSContext configuration if SSL is enabled with Ambassador and returns empty string if SSL is not enabled
+func getAmbassadorTLSContextConfig(mlDep *machinelearningv1.SeldonDeployment, p *machinelearningv1.PredictorSpec, addNamespace bool) (string, error) {
+
+	if p.SSL == nil || len(p.SSL.CertSecretName) <= 0 {
```

Comment:
nit: we can move the "empty" check to a helper function, like `isEmptyExplainer`. That way we also avoid problems in the future if we change the criteria to consider it empty.

https://github.com/SeldonIO/seldon-core/blob/47380b578b9cdc5565c3cfd9e3796d6ecc4ddada/operator/controllers/controller_utils.go#L82-L86

Reply:
Yes definitely, want to actually move it to a function that checks whether ssl is enabled mainly as we may want to introduce further integration with certmanager (as the PR before actually created the cert-manager certificates in the operator but we decided to remove it in this version)